### DISCUSSION TITLE
/DBC-Wiki/onchain-guide/rent-machine.html

### DISCUSSION THREAD

**Discussion by giscus** [DeepBrainChain/DBC-Wiki#36] [Announcements]
# /DBC-Wiki/onchain-guide/rent-machine.html

欢迎来到 DBC-Wiki

https://deepbrainchain.github.io/DBC-Wiki/onchain-guide/rent-machine.html

**Comment by CHENyuuuuuuu** [comment]
步骤二：租用链上机器，这个在哪呀
